client/components/ScenariosList/ScenarioEntries.jsx

```diff
@@ -6,15 +6,30 @@ import ScenarioCard from './ScenarioCard';
 const ScenarioEntries = ({ scenarios, isLoggedIn }) => {
     if (!scenarios.length) return null;
 
-    return scenarios.map(scenario => {
-        return (
+    return scenarios.reduce((accum, scenario) => {
+        const { status, user_is_author: isAuthor } = scenario;
+
+        // This scenario status is "draft", to see it:
+        //  - user must be logged in
+        //  - user must be the author
+        if (status === 1 && (!isLoggedIn || !isAuthor)) {
```

Comment:
Maybe for these status numbers we can use some constants like `const IS_DRAFT = 1`? If we do that and you agree, I think we can skip the comments.

Reply:
Love it. I'll update that now.